src/__tests__/select.test.ts

```diff
@@ -334,10 +334,8 @@ describe(`select`, () => {
   });
 
   it(`should select IN with array`, () => {
-    const query = db
-      .select(db.foo.id)
-      .from(db.foo)
-      .where(db.foo.name.in([`A`, `B`, `C`]));
+    const array = ['A', 'B', 'C'] as const; // Make sure you can pass read-only arrays.
+    const query = db.select(db.foo.id).from(db.foo).where(db.foo.name.in(array));
```

Comment:
Should we write a new test for this to test both read-only and regular arrays?

Reply:
Since `Readonly` in TS just makes something accept more values, I don't think it's worth writing a test for the less constraining version of the value.